FrontEnd/src/components/SearchPage/search_field/Text.css

```diff
@@ -0,0 +1,49 @@
+.new-companies-main {
+    width: 1304px;
```

Comment:
Should it be 100%? What if the screen is smaller? We will have a horizontal scroll. We agreed to support at least 1024px screen width

Reply:
We had this issue on conversetion with developers.
We chose this with in start of the project
After we will finish functionalitty on front and  back we will fix it
thank you for your consideration